src/ui/components/Passport/Passport.tsx

```diff
@@ -29,6 +30,10 @@ type PrimaryPanelName = "events" | "cypress" | string;
 const stepNames = ["step-one", "step-two", "step-three", "step-four"] as const;
 
 const Passport = (props: PropsFromRedux) => {
+  const [showTestsuitesPassportWelcome, setshowTestsuitesPassportWelcome] = useState(
+    localStorage.getItem("TestsuitesPassportWelcome") !== "true"
```

Comment:
:hand: I think we want to avoid using `localStorage` directly at this point, now that Brian has moved all of our settings into the new `UserData` system.

Can you try defining the setting with that system instead? I believe it also has a hook you can use to retrieve and set the value.

Reply:
Roger Roger